lib/node_modules/@stdlib/string/next-grapheme-cluster-break/README.md

```diff
@@ -0,0 +1,185 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2020 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# nextGraphemeClusterBreak
+
+> Return the next extended grapheme cluster break in a string after a specified position.
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var nextGraphemeClusterBreak = require( '@stdlib/string/next-grapheme-cluster-break' );
+```
+
+#### nextGraphemeClusterBreak( string\[, fromIndex] )
+
+Returns the next extended grapheme cluster break in a string after a specified position.
+
+```javascript
+var out = nextGraphemeClusterBreak( 'last man standing' );
+// returns 1
+```
+
+By default, the first extended grapheme cluster break in the string is returned. For the next extended grapheme cluster break after a specified position in the string, provide a `fromIndex`.
+
+```javascript
+var str = nextGraphemeClusterBreak( 'last man standing', 4 );
+// returns 5
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- Package usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+## Notes
+
+-   If `string` is an empty string, the function returns `-1` irrespective of `fromIndex`.
+-   If there is no extended grapheme cluster break after `fromIndex`, the function returns `-1`.
+-   Note that `fromIndex` does **not** refer to a visual character position, but to an index in the ordered sequence of [UTF-16][utf-16] code units.
```

Comment:
**Aside**: we may want to consider, if we don't already have such functionality, one or more string utilities (as separate packages) for indexing, and maybe iterating over, visual characters. I believe such utilities would build on the current work.